denbi/perun/endpoint.py

```diff
@@ -283,19 +298,27 @@ def __import_dpcc_projectdata__(self, json_obj):
                     if set(project['members']) != set(members) or \
                             project['name'] != name or \
                             'description' in project and project['description'] != description:
-                        self.logging.info("Updating project %s with {%s}", perun_id, ','.join(members))
+                        # Update project ...
                         self.keystone.projects_update(perun_id, members)
+                        # ... and log to update logger
+                        self.log2.info("Update project [%s,%s] with (%s)", name, perun_id,",".join(members))
 
                     # check for quotas and update it if possible
                     if self.support_quotas:
-                        self._set_quotas(project, dpcc_project)
-
+                        if self.read_only:
+                            self.log.info("Not setting quotas for new project [%s,%s] in  readonly mode", name, perun_id)
+                        else:
+                            self._set_quotas(project, dpcc_project)
                 else:
-                    self.logging.info("Creating project %s with name=%s members={%s}", perun_id, name, ','.join(members))
+                    # create project ...
                     project = self.keystone.projects_create(perun_id, name=name, description=description, members=members)
+                    # ... and log to update logger
+                    self.log2.info("Create project [%s,%s] with (%s)", name, perun_id,",".join(members))
+
+                    # check for quotas and update it if possible
                     if self.support_quotas:
                         if self.read_only:
-                            self.logging.info("Not setting quotas for new project %s, readonly mode", name)
+                            self.log.info("Not setting quotas for new project [%s,%s] in  readonly mode", name, perun_id)
```

Comment:
When trying to give information about which project is specified, try specifiying it in the beginning.
For example:
Project [dsafds45-fdsf32-fdf2]: Not setting quotas....

Sometimes the project_id's are mixed between the left side and right side of the log line. Making it more uniform would make the logfile more readable.

Reply:
Good point! I will change this.